packages/components/number-input/src/number-input.tsx

```diff
@@ -144,8 +144,9 @@ export type UseNumberInputProps = UseFormControlProps<HTMLInputElement> &
   }
 
 export const useNumberInput = (props: UseNumberInputProps = {}) => {
+  const id = useId()
   const {
-    id,
+    id: _id,
```

Comment:
Where is `_id` used?
Where `id` was originally used, it should be replaced with `_id`.

Reply:
@cidkumagai 
Sorry.
Please change `_id` to `idProp`.

Reply:
fixed now.
please re-review